site/ts/sql-toolbox.js

```diff
@@ -129,6 +171,45 @@ export function init() {
                     : 'block';
         });
     });
+    window.addEventListener('DOMContentLoaded', () => {
```

Comment:
`init()` is only called on `DOMContentLoaded` so why set another nested listener?